src/Contract/Address.purs

```diff
@@ -141,8 +145,13 @@ getWalletAddress = head <$> getWalletAddresses
 -- | Get all the `Address`es of the browser wallet.
 getWalletAddresses
   :: forall (r :: Row Type). Contract r (Array Address)
-getWalletAddresses = do
-  addresses <- wrapContract QueryM.getWalletAddresses
+getWalletAddresses = getWalletAddressesPaginated Nothing
+
+-- | Get all the `Address`es of the browser wallet with pagination.
+getWalletAddressesPaginated
+  :: Maybe Paginate -> forall (r :: Row Type). Contract r (Array Address)
```

Comment:
The paginated version should require a `Paginate` argument (remove `Maybe`)

Reply:
getWalletAddressesPaginated - is more general version used in `getWalletAddresses`.

Reply:
@uhbif19 yep, please do